packages/react-router/__tests__/router/utils/data-router-setup.ts

```diff
@@ -225,37 +222,11 @@ export function setup({
       let enhancedRoute: AgnosticDataRouteObject = {
         unstable_middleware: undefined,
         ...r,
-        lazy: undefined,
         loader: undefined,
         action: undefined,
         children: undefined,
         id: r.id || `route-${guid++}`,
       };
-      if (r.lazy) {
-        // @ts-expect-error
-        enhancedRoute.lazy = (args) => {
```

Comment:
I've removed the lazy route helpers since the new approach to deduping means that it's more of a top-level concern within the test, not tied to individual navigations. This simplified the tests quite a bit.

Reply:
@brophdawg11 Sounds good. That was exactly my thinking too.